src/test/java/org/sagebionetworks/web/client/SynapseJavascriptClientTest.java

```diff
@@ -225,9 +225,7 @@ public void before() throws RequestException {
     when(mockJsniUtils.getAccessTokenCookieUrl())
       .thenReturn(SESSION_COOKIE_URL);
 
-    when(
-      mockRequestBuilder.sendRequest(anyString(), any(RequestCallback.class))
-    )
+    when(mockRequestBuilder.sendRequest(any(), any(RequestCallback.class)))
```

Comment:
For nearly all of these test changes, the mock was called where the matcher received `null`. In Mockito v1, `anyString()` matched null. In Mockito v2, it does not. Replace it with `any()`.

Reply:
Ah, now I understand why my newly checked in test with anyLong() may fail :)